lib/cli.rb

```diff
@@ -618,21 +618,20 @@ def self.parse_inspect_command_options(host, options)
   desc "Inspect container image"
   long_desc <<-LONGDESC
     Inspect container image and generate system descripton from inspected data.
+    Right now we only support docker images. Further types will be supported in the future as well.
 
     Multiple scopes can be passed as comma-separated list. If no specific scopes
     are given, all scopes are inspected.
 
     Available scopes: #{AVAILABLE_SCOPE_LIST}
   LONGDESC
-  arg "IMAGENAME"
+  arg "<IMAGENAME|IMAGEID>"
```

Comment:
Are the `<>` following some convention? We aren't using them in the other commands.